main/traffic.go

```diff
@@ -835,6 +845,17 @@ func updateDemoTraffic(icao uint32, tail string, relAlt float32, gs float64, off
 	if ti.Addr_type == 1 {         //reserved value
 		ti.Addr_type = 0
 	}
+
+	if ti.Addr_type == 0 {
+		ti.TargetType = TARGET_TYPE_ADSB
+	}
+	if ti.Addr_type == 2 {
```

Comment:
these two checks can be done with an "if else" check 

I guess if ti.Addr_type isn't 0 or 2 or 3 then the default for ti.TargetType is TARGET_TYPE_MODE_S (TARGET_TYPE_MODE_S is 0 which is what ti.TargetType is initialized to). It'd be clearer if the last check was an else that explicitly defaults to the desired value. E.g.

        if ti.Addr_type == 0 {
		ti.TargetType = TARGET_TYPE_ADSB
	} else if ti.Addr_type == 2 {
		ti.TargetType = TARGET_TYPE_ADSR
	} else if ti.Addr_type == 3 {
		ti.TargetType = TARGET_TYPE_TISB
	} else {
                ti.TargetType = TARGET_TYPE_MODE_S // or should this be something else
        }